data/containers/registries.conf

```diff
@@ -1,20 +1,24 @@
 # For more information on this configuration file, see containers-registries.conf(5).
-#
+
 # Registries to search for images that are not fully-qualified.
-# i.e. foobar.com/my_image:latest vs my_image:latest
-[registries.search]
-registries = ["registry.opensuse.org", "docker.io"]
+unqualified-search-registries = ["registry.opensuse.org", "registry.suse.com", "docker.io"]
+
+[[registry]]
+location = "registry.opensuse.org"
+
+[[registry]]
+location = "docker.io"
+
+[[registry]]
+prefix = "localhost"
+location = "localhost:5000"
+insecure = true
 
-# Registries that do not use TLS when pulling images or uses self-signed
-# certificates.
-[registries.insecure]
-registries = ["localhost:5000", "registry.suse.de", "REGISTRY"]
+[[registry]]
+location = "registry.suse.de"
+insecure = true
```

Comment:
Maybe interchange those?

Reply:
What do you mean with interchange those? I kept the same order as in the original file for those registries.

Reply:
I ment order. Than it's OK.